docs/tutorials/ui-customization.md

```diff
@@ -99,11 +99,12 @@ The following buttons can be added to the overflow menu:
   supports AirPlay.
 * remote: adds a button that opens a Remote Playback dialog. The button is visible only if the
   browser supports Remote Playback API.
-* Statistics: adds a button that displays statistics of the video.
+* statistics: adds a button that displays statistics of the video.
 * recenter_vr: adds a button that recenter the VR view to the initial view. The button is visible
   only if playing a VR content.
 * toggle_stereoscopic: adds a button that toggle between monoscopic and stereoscopic. The button
   is visible only if playing a VR content.
+* ad_statistics: adds a button that displays ad statistics of the video.
```

Comment:
You should also add `ad_statistics` (and fix the capitalization on `statistics`) in the listing under `#### Replacing the default context menu`, since it looks like your new button can also be added to the context menu.

...Perhaps we should come up with some way of generating this automatically, since it's such an annoying thing to keep in sync. It was simpler to maintain this doc when we didn't have all of these different menus! Well, another time.

Reply:
Yes, maintaining this is a pain, we should change it in the future... :(